omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/visit/RetrospectiveVisitFragmentController.java

```diff
@@ -31,31 +31,30 @@ public Object create(@SpringBean("adtService") AdtService adtService,
                                        @RequestParam(value = "stopDate", required = false) Date stopDate,
                                        HttpServletRequest request, UiUtils ui) {
 
-        // if no stop date, set it to start date
-        if (stopDate == null) {
```

Comment:
What was the motivation for removing this?

Reply:
This is because with this enhancement we can allow a user to pass a null stopDate which will make that retrospective visit an active visit....this code was instead forcing any such action to default the end date to startDate